app/src/actions/map.js

```diff
@@ -112,85 +119,178 @@ export function setCenter(center) {
     payload: center
   };
 }
-/*
- ** CartoDB layers:
- ** MPA
- ** EEZ
- ** High Seas Pockets
- ** RFMOs
+
+/**
+ * Retrieve the workspace according to its ID and sets the zoom and
+ * the center of the map, the timeline dates and the available layers
+ *
+ * @export getWorkspace
+ * @param {string} workspaceId - workspace's ID to load
+ * @returns {object}
  */
-export function getWorkspace(workspace) {
+export function getWorkspace(workspaceId) {
   return (dispatch, getState) => {
     const state = getState();
 
-    let path = '/workspace.json';
-    if (state.user.token) {
-      path = '/workspace-logged.json';
-    }
+    // If the user isn't logged, we load a local workspace
+    let url = '/workspace.json';
 
-    if (!!~[1, 2].indexOf(+workspace)) {
-      path = `/workspace-${workspace}.json`;
+    if (state.user.token && workspaceId) {
+      url = `${API_URL}/workspaces/${workspaceId}`;
     }
 
-    fetch(path, {
-      method: 'GET',
+    fetch(url, {
```

Comment:
Are we using a polyfill for `fetch`? In many other places we have `import 'whatwg-fetch';`